.gitignore

```diff
@@ -1,5 +1,5 @@
+.idea
 build
-dist/klaro.js
```

Comment:
`dist/klaro.js` is a build artifact. It shouldn't be included in the repository.
All other files in `dist` are only there because they are static files for the website: https://klaro.kiprotect.com/